job-scraper/job_scraper.py

```diff
@@ -0,0 +1,18 @@
+import requests
+
+
+class JobScraper:
+
+    @staticmethod
+    def get_url(URL):
+        r = requests.get(url=URL)
+        data = r.json()
+        return data
+
+    @staticmethod
+    def find_python_job(data, value):
+        for k, v in data.items():
+            if type(v) == str and value in v.lower():
+                return True, data
+            else:
```

Comment:
W tym przypadku tego elsa możesz odrzucić.`pass` w takim przypadku zwraca `None` to samo osiągniesz nie robiąc tego else'a - automatycznie też zostanie zwrócony `None`.